<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Окно Админестратора</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1091</width>
      <height>711</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="Tab">
        <attribute name="title">
         <string>Главная</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>1081</width>
           <height>501</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="Income_for_mouth">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Доход  в этом месяце&lt;/span&gt;:1111&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="count_tickest">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Количество проданных билетов: &lt;/span&gt;100&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Подробная аналитика</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="user_name">
            <property name="font">
             <font>
              <family>Nirmala UI</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Привет Пользователь</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Управление сеансами</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>1071</width>
           <height>661</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QLineEdit" name="search_l_2">
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="1" rowspan="9">
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2" rowspan="9">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="session_name_2">
              <property name="font">
               <font>
                <family>Nirmala UI</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Выберите сеанс</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="time_2">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="picture_2">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextBrowser" name="about_2">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>Редоктровать</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QDateEdit" name="dateEdit"/>
          </item>
          <item row="5" column="0" rowspan="4">
           <widget class="QListWidget" name="list_of_sessions_2">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>450</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>26</height>
             </size>
            </property>
            <property name="text">
             <string>Поиск</string>
            </property>
            <property name="indent">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QToolButton" name="add_btn">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>Сеансы</string>
            </property>
            <property name="indent">
             <number>3</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="sessions_control"/>
    <addaction name="users_control"/>
    <addaction name="separator"/>
    <addaction name="action_exit_2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Редактировать пользователя</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>Выйти</string>
   </property>
  </action>
  <action name="set_user_inf">
   <property name="text">
    <string>Редактировать</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Выйти</string>
   </property>
  </action>
  <action name="sessions_control">
   <property name="text">
    <string>Управление сеансами</string>
   </property>
  </action>
  <action name="users_control">
   <property name="text">
    <string>Управление пользователями</string>
   </property>
  </action>
  <action name="action_exit_2">
   <property name="text">
    <string>Выйти из профиля</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
